src/grimp/adaptors/caching.py

```diff
@@ -201,9 +212,7 @@ def _read_data_map_file(self) -> Dict[Module, Set[DirectImport]]:
             logger.warning(f"Could not use corrupt cache file {data_cache_filename}.")
             return {}
 
-        primitives_map: PrimitiveFormat = self._to_primitives_data_map(
-            deserialized_json
-        )
+        primitives_map: PrimitiveFormat = self._to_primitives_data_map(deserialized_json)
```

Comment:
There are lots of examples of changes to the formatting, which is presumably due to a mismatch between your editor line length settings and mine. I'd prefer not to include these formatting changes in this PR unless there is a good reason.

I thought [Black](https://pypi.org/project/black/) was already set up in CI but turns out it isn't. I've made [a ticket](https://github.com/seddonym/grimp/issues/131) to look into this which I'll look at in the next few days.